tests/active_features/auth.feature

```diff
@@ -73,13 +73,29 @@ Feature: Authorization command
     [[username]]
     """
 
-  @vcr cli_session-dump
-  Scenario: Dumping a session
+  @vcr auth_whoami_id
+  Scenario: Check Which User I Am by id
```

Comment:
Everything in this sentence is capitalized except the one word that needs to be.